cmd/deploy/deploy.go

```diff
@@ -160,6 +161,7 @@ func Deploy(ctx context.Context) *cobra.Command {
 	cmd.Flags().StringVar(&options.Name, "name", "", "application name")
 	cmd.Flags().StringVarP(&options.ManifestPath, "file", "f", "", "path to the manifest file")
 	cmd.Flags().StringArrayVarP(&options.Variables, "var", "v", []string{}, "set a variable (can be set more than once)")
+	cmd.Flags().StringVarP(&options.OutputMode, "output", "", "tty", "show plain/tty deploy output")
```

Comment:
If we would deploy first the installer using the `okteto deploy` command and then, we merge this and update the cloudbin image, the installer could use tty output, which I think is not the best. We have to coordinate how to do this

Reply:
talked offline. We'll change the installer to send the right output option when we update the CLI in the backend